geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedAckRegionCCEDUnitTest.java

```diff
@@ -17,16 +17,16 @@
 
 import static org.apache.geode.distributed.ConfigurationProperties.CONSERVE_SOCKETS;
 import static org.apache.geode.distributed.ConfigurationProperties.DISTRIBUTED_SYSTEM_ID;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-import static org.junit.Assume.assumeTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
```

Comment:
Take this import out and you'll find all the uses of `fail()`. Just let the test runner handle errors form any unexpected exceptions.

Reply:
GEODE-5669 has been submitted to address usages of fail.

Reply:
a new task has been created to address this issue, mentioned elsewhere in these discussions.